snuba/cli/subscriptions_scheduler_executor.py

```diff
@@ -70,6 +71,11 @@
     help="Skip scheduling if timestamp is beyond this threshold compared to the system time",
 )
 @click.option("--log-level", help="Logging level to use.")
+@click.option(
+    "--override-partition-mode",
```

Comment:
Nit: Instead of calling this override-partition-mode, would it make more sense to call it `--scheduling-mode`. What this means is that if a parameter is provided from the CLI then use that instead of the default configuration. That would make it more explicit what scheduling mode the consumer is running on.

Reply:
Sure, ok